docs/billing.md

```diff
@@ -1,5 +1,51 @@
 # Billing
 
+Before diving into billing let's do a recap of some parts of important parts of the system.
+
+Note: During the rest of the documentation we'll consider ChainA as source chain and chainB as destinationChain.
```

Comment:
If you rename `chainA` to `sourceChain` and `chainB` to `destinationChain` you can skip the note

Reply:
+1, source + dest is typically easier to understand

Reply:
I purposefully didn't use source and dest so that when I reach the part where each source (which is dest) adds other sources' gas prices. I'll finish the doc and if you still think it's better to rename I'll do so.